polynomials.go

```diff
@@ -160,11 +161,20 @@ const randPolMaxTries = 1e6
 // "Fingerprinting by Random Polynomials", page 4. If no polynomial could be
 // found in one million tries, an error is returned.
 func RandomPolynomial() (Pol, error) {
```

Comment:
The documentation here should be changed to something like "Returns a random Polynomial by calling `DerivePolynomial()` with the source set to `rand.Reader`". What do you think?